text/0000-project-groups.md

```diff
@@ -0,0 +1,391 @@
+regular triage.
+
+## Community Groups
+
+Community Groups are groups of individuals who want to work and collaborate on a
+specific topic, without going through the formal process of a working group.
+Community groups are encouraged to adopt the structure of working or projects
+groups laid, but are free create and experiment with their own structure. As
+such community groups are not officially endorsed by The Rust Programming
+Language Organisation.
+
+## Archival
+
+The archival process of a group involves communicating the wind down to the
+community and the archival or transfer of ownership of the relevant projects.
+As well archiving any chat channels hosted by the Rust project.
+
+### Retrospectives
+
+While this RFC attempts to address some of the current organisational problems
+within the organisation, it also doesn't believe that this RFC will be a panacea
+to those problems or that we won't encounter more in the future. As part of
+that, we'd also like to introduce performing retrospectives with groups, once
+significant time has past or the group has been finished it's project.
+
+This would involve a discussion between the members of the group, and ideally
+their parent team and the Governance working group. The retrospective should
+produce a public blog post on the Inside Rust blog, however any feedback a
+member has that they would want to keep private would be omitted.
+
+The blog post should try to cover the output of the group, such as RFCs or
+projects, as well what the group thought worked and importantly what
+didn't work. This should help us iterate on this initial RFC and help us find
+and address issues that come up in the process.
+
+# Drawbacks
+
+- This will add organisational overhead to maintain group repositories
+  and documents.
+- There's a lot of inertia around the Working Group terminology, and switching
+  to new terminology will likely also cause some confusion, though hopefully
+  only in the short term.
+
+# Future Work
+
+- Ideally we'd prefer if every team obtained consensus to form groups through
+  RFCs, as they an open process that allows us to easily keep track of
+  decisions. However we recognise that the current RFC process is maybe too
+  heavyweight for some teams currently. We're currently looking how we can
+  simplify some of this process, see [wg-governance#38] for further information.
+
+[wg-governance#38]: https://github.com/rust-lang/wg-governance/issues/38
+
+# Unresolved questions
+
+[unresolved-questions]: #unresolved-questions
+
+- The term _"shepherd"_ term has been used extensively in the Rust project and
+  the community to describe leaders of teams however there hasn't ever been a
+  strict definition and this could come with different expectations of what is
+  expected from a shepherd. This RFC does not attempt to define this, however
+  there are few resources that are helpful to understanding the terminology.
```

Comment:
I would prefer to resolve this by not using the term. To my recollection, the uses of the term in this RFC can mostly be replaced with "lead" wrt to project groups.

Reply:
I agree that shepherd and lead are quite close and maybe identical. I could be convinced to just adopt lead, but I think that we haven't exactly done a good job of defining the role of a **lead** either, so I'm not sure we've solved the problem. (But maybe we should work on writing those things out in a more evolutionary way, on forge.)